crates/re_sdk/src/recording_stream.rs

```diff
@@ -31,6 +36,10 @@ pub enum RecordingStreamError {
         name: &'static str,
         err: Box<dyn std::error::Error + Send + Sync>,
     },
+
+    #[cfg(feature = "web_viewer")]
+    #[error(transparent)]
+    Other(#[from] anyhow::Error),
```

Comment:
Imho making this explicit would be nice otherwise it might be used for non-web things by accident as long as the web_viewer feature is on

```suggestion
    WebSink(#[from] anyhow::Error),
```